serve/source.go

```diff
@@ -63,6 +64,13 @@ func newCmdSourceServe(source *sourceServe) *cobra.Command {
 	var noSentry bool
 	logLevel := newEnum([]string{"trace", "debug", "info", "warn", "error"}, "info")
 	logFormat := newEnum([]string{"text", "json"}, "text")
+	telemetryLevel := newEnum([]string{"none", "errors", "stats", "all"}, "all")
```

Comment:
Im abit confused. we don't have telemetry on the plugin side. Can we just add no-sentry? or just have `none/errors`

Reply:
Actually seems like there is already a `no-sentry` flag, so maybe let just use that?

Reply:
Yes, this PR is re-using the existing `no-sentry` flag, and this isn't adding a new flag. But what is changing is that the plugins now respect `CQ_TELEMETRY_LEVEL`, so that developers who have that set in their environment don't accidentally send crash reports if they forget to run a plugin with the `--no-sentry` flag.